build.gradle

```diff
@@ -27,21 +28,22 @@ buildscript {
   ext.isCi = "true" == System.getenv('CI')
 
   repositories {
-    jcenter()
     google()
+    maven { url 'https://plugins.gradle.org/m2/' }
   }
 
   dependencies {
     classpath deps.androidPlugin
     classpath 'com.github.ben-manes:gradle-versions-plugin:0.17.0'
+    classpath 'net.ltgt.gradle:gradle-errorprone-plugin:0.0.13'
   }
 }
 
 apply plugin: 'com.github.ben-manes.versions'
 
 subprojects {
   repositories {
-    jcenter()
+    mavenCentral()
```

Comment:
does the plugin require this?  I thought plugins only require buildscript dependencies.

Reply:
if not, let's remove.  else move under `google()`.

Reply:
Unrelated change, yeah.

Reply:
okay, looking on my computer now...
this change isn't related to the plugin. i just figured we prefer mc over jcenter.
we need one of them for the dependencies like OkHttp, etc.
`google()` is below.

what are you asking?

Reply:
understood.  when I originally reviewed this (on mobile), I thought this diff related to the subject line of this PR ("errorprone").